docs/guides/5-custom-functions.md

```diff
@@ -84,7 +84,38 @@ rules:
 Where the function `functions/equals.js` might look like:
 
 ```js
-import { createRulesetFunction } from '@stoplight/spectral-core';
+import { createRulesetFunction } from "@stoplight/spectral-core";
+
+export default createRulesetFunction(
+  {
+    input: null,
+    options: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        value: true,
+      },
+      required: ["value"],
+    },
+  },
+  (input, options) => {
+    const { value } = options;
+
+    if (targetVal !== value) {
+      return [
+        {
+          message: `Value must equal ${value}.`,
+        },
+      ];
+    }
+  },
+);
+```
+
+You can alo name the custom function by importing `IFunctionResult`. This can help debug any errors as the function name will be printed out in any error messages:
```

Comment:
```suggestion
You can also name the custom function by importing `IFunctionResult`. This can help debug any errors as the function name will be printed out in any error messages:
```